src/handbook/peopleops/hiring.md

```diff
@@ -154,3 +154,55 @@ for the open position, take the job posting offline. Candidates applying for a j
+
+Once you have discussed your concerns with your manager and the People Ops team and determined that resigning from FlowForge is the only option, please notify both your manager and the People Ops team of your intention to resign via email. Afterward, we will commence a discussion to establish the steps required to ensure a seamless handover.
+
+### Involuntary Resignation
+
+When a team member is being let go, it is usually due to either performance issues or a change in the company's needs that no longer justifies their role. If the decision is due to performance issues, we will have already provided feedback and given the team member an opportunity to improve. Unfortunately, if the issues cannot be resolved, termination of employment may be necessary.In most cases, we will ask the team member to stop working immediately. The final payment will be made according to the terms of their employment contract.
+
+### Communicating Departures
+
+When a team member voluntarily departs, we may ask if they would like to share their plans with the team. If the departure is involuntary, we will strive to maintain transparency regarding the reasons for the departure, while still respecting the individual's privacy.
+
+Please note that in some instances, FlowForge may be unable to provide additional context surrounding an employee's departure.
+
+## Offboarding process
+
+We will send an email to the departing employee covering the following points:
+
+Final payment
+Vested share options
+Return of company property
+Reimbursement of business expenses
+
+At FlowForge, we conduct Exit Interviews for employees who choose to leave the company voluntarily. This provides us with valuable feedback on their experience, reasons for leaving, and areas for improvement. The People Ops manager schedules these interviews during the employee's last week to ensure they have an opportunity to share their insights. Our aim is to use this feedback to enhance our workplace culture and practices.
+
+### Offboarding Issue
+
+The People Ops manager will create a new offboarding Issue for each leaver using the template maintained on GitHub, which can be viewed https://github.com/flowforge/admin/issues/166
+
+### Final Pay
+
+Final pay for employees leaving FlowForge will be determined based on their length of service and the reason for leaving. If the offboarding is voluntary, the employee will be paid up until their last day and any unused vacation pay will be paid out based on assuming the employee would have taken 25 days of vacation (as we offer unlimited PTO).
+
+If offboarding is involuntary due to performance or business changes, severance pay will be provided at the sole discretion of FlowForge management.
```

Comment:
Much of this section is based on what Deel dictates and where the employee is located, what about:

```suggestion
### Final Pay

Final pay consists of the statutory minimal payment dictated by our payroll provider Deel, and an optional extra severance package. Such a package might be dependent on on business requirements, standing of the employee, among other factors.
```